src/frontend/src/pages/AdminToolsPage/AdminToolsSlackUpcomingDeadlines.tsx

```diff
@@ -66,7 +70,17 @@ const AdminToolsSlackUpcomingDeadlines: React.FC = () => {
             Send
           </NERButton>
         </Grid>
+        <Grid item>
+          <NERButton
+            variant="contained"
+            disabled={!isAdmin(user.role) || disableButton || isLoading}
```

Comment:
you don't need disableButton or isLoading here, they're specifically for slack upcoming deadlines

Reply:
honestly, I would take this whole feature out of the modal and just put it in it's own pageblock below the slack upcoming deadlines pageblock